apps/meteor/client/views/room/hooks/useRetentionPolicy.ts

```diff
@@ -89,11 +89,11 @@ export const useRetentionPolicy = (
 		filesOnly: useSetting('RetentionPolicy_FilesOnly') as boolean,
 		doNotPrunePinned: useSetting('RetentionPolicy_DoNotPrunePinned') as boolean,
 		appliesToChannels: useSetting('RetentionPolicy_AppliesToChannels') as boolean,
-		maxAgeChannels: useSetting('RetentionPolicy_MaxAge_Channels') as number,
+		maxAgeChannels: useSetting('RetentionPolicy_TTL_Channels') as number,
```

Comment:
The casting here is necessary since useSetting even after passing the type parameter says it could be undefined. I won't fix this now because I'd have to refactor the entire file.